### PR TITLE
NO-ISSUE: Document how to add OCM capabilities

### PR DESCRIPTION
**Pull request by jhernand**
This patch adds a section to the `ocm-integration.md` document explaining how to add capabilities to the OCM accounts management service.

## List all the issues related to this PR

- [ ] New Feature 
- [ ] Enhancement 
- [ ] Bug fix
- [ ] Tests
- [x] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [ ] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Checklist

- [x] Title and description added to both, commit and PR.
- [x] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [x] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: docs/ocm-integration.md [hunk 1]
```diff
@@ -242,3 +242,117 @@ ocm get subscriptions --parameter search="status = 'Active'"
...
+        CapabilityUseRosaPaidAMI,
+```
+
+That needs to be merged, and then the new version of AMS pushed to the
+corresponding environment (integration, stage and eventually production). The
+SD-B team is responsible for that, ping the `@sd-b-team` in the
+[#service-development-b](https://coreos.slack.com/archives/CBDNMS43V) channel
+in Slack.
+
+The second step is to enable the capability for the organizations that you
```
Comment (gamli75) [inline docs/ocm-integration.md:292]: can we enable the capability for specific email domains such as: nokia.com?

File: docs/ocm-integration.md [hunk 2]
```diff
@@ -242,3 +242,117 @@ ocm get subscriptions --parameter search="status = 'Active'"
...
+sts_ocm_role
+capability.organization.bare_metal_installer_multiarch
+capability.organization.hypershift
+capability.cluster.subscribed_ocp
+moa_entitlement_expires_at
+```
+
+Chances are that you will not have permission to enable or disable
+capabilities, specially in the production environment. In that case open a
+ticket in the [SDB](https://issues.redhat.com/projects/SDB) board requesting
```
[the remark this answers is not in the document]
Comment (jhernand) [reply docs/ocm-integration.md]: Done, used https://issues.redhat.com/browse/SDB-3211 as the example.

File: docs/ocm-integration.md (hunk 1 above)
Comment (jhernand) [reply docs/ocm-integration.md:292]: No, the OCM accouns management system doesn't support that, capabilities need to be enabled by organization or by account.

File: docs/ocm-integration.md [hunk 3]
```diff
@@ -242,3 +242,141 @@ ocm get subscriptions --parameter search="status = 'Active'"
...
+```shell
+$ ocm get /api/accounts_mgmt/v1/organizations/2GOuxr72XcFbrWvyA9rS9McOUjO/labels | jq -r '.items[] | select(.type == "Organization") | .key'
+sts_ocm_role
+capability.organization.bare_metal_installer_multiarch
+capability.organization.hypershift
+capability.cluster.subscribed_ocp
+moa_entitlement_expires_at
+```
+
+Chances are that you will not have permission to enable or disable
```
Comment (gamli75) [inline docs/ocm-integration.md:338]: regarding the permission, please add that the user should have the BareMetalInstallerAdmin for integration & staging, and for production we should open a ticket

File: docs/ocm-integration.md [hunk 4]
```diff
@@ -242,3 +242,149 @@ ocm get subscriptions --parameter search="status = 'Active'"
...
+
+```shell
+$ ocm login --token ... --url stage
+```
+
+The default, when the `--url` option isn't provided is to use the production
+environment.
+
+If you need to do this very often in the integration or stage environments, and
+without asking the SDB team, you can request the `BareMetalInstallerAdmin` role
```
Comment (osherdp) [inline docs/ocm-integration.md:385]: add ``- UHCSupport:`` and it's a complete solution
(https://gitlab.cee.redhat.com/service/ocm-resources/-/merge_requests/2725)

**Review by osherdp — CHANGES_REQUESTED** [review]

File: docs/ocm-integration.md (hunk 4 above)
Comment (jhernand) [reply docs/ocm-integration.md]: Done.

**Review by osherdp — APPROVED** [review]

**Pull request merged**